### ISSUE TITLE
[ts] onClearText does not exist in SearchBar properties

### ISSUE THREAD

**Issue by changx03** [react-native-training/react-native-elements#757]
```
"react-native-elements": "^0.18.4"
"@types/react-native-elements": "^0.16.4"
```

TypeScript cannot find onClearText in SearchBar's props. 

```
            <SearchBar
              lightTheme
              onChangeText={text => { AppStore.searchEmployee(text); }}
              onClearText={() => { AppStore.loadFullEmployeeList(); }}
              placeholder="Search..."
              clearIcon={{name: 'clear'}}
            />
```

The functionality seems here, probably due to missing typings.

![capture](https://user-images.githubusercontent.com/9165455/33574683-d68c54a0-d99e-11e7-9b30-393ec34bc2db.PNG)
with error:

> Property 'onClearText' does not exist on type 'IntrinsicAttributes & IntrinsicClassAttributes<SearchBar> & Readonly<{ children?: ReactNode; }> &...'.

**Comment by iRoachie** [comment]
Thanks for reporting! Will make the relevant pull requests

**Comment by iRoachie** [comment]
Added in https://github.com/react-native-training/react-native-elements/pull/751/commits/1e06ae893ccdb316d49723c06d58136d3d10dd08. Will be available next release.

**Comment by xavier-villelegier** [comment]
@changx03 Fixed in `v0.18.5`!
